pr_insight/algo/utils.py

```
@@ -910,7 +914,7 @@
         if rate_limit_info.get('message') == 'Rate limiting is not enabled.':  # for github enterprise
             return {'resources': {}}
         response.raise_for_status()  # Check for HTTP errors
-    except Exception:
+    except:  # retry
```

Ruff

Do not use bare `except`

(E722)